src-cpp/app/modules/main/wallet/accounts/controller.cpp

```diff
@@ -22,4 +22,45 @@ QList<Wallets::WalletAccountDto> Controller::getWalletAccounts()
 
     return wallet_accounts;
 }
+
+QString Controller::generateNewAccount(QString password, QString accountName, QString color)
```

Comment:
and again `const QString&`should be everywhere

Reply:
can't be as I am creating a QString in local scope now to handle the nullptr use case.

Reply:
but what about `QString password, QString accountName, QString color`?

Reply:
done!